module_1/assignment2_2.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+</head>
+<body>
+
+  <script>
+  document.getElementById("test").innerHTML = "Goodbye World";
```

Comment:
Please add comment what happen when we add script tag at the start and end of body tag